backend/api/models.py

```diff
@@ -171,3 +171,13 @@ class Stats(models.Model):
     confidence_low = models.FloatField(blank=True, null=True)
     confidence_high = models.FloatField(blank=True, null=True)
     confidence_level = models.FloatField(blank=True, null=True)
+
+
+class Uptake(models.Model):
```

Comment:
Can we call this Enrollment instead?  In Shield land it's called enrollment and I'm trying to use the Shield language as much as possible.

Reply:
I do like that better. Consistency FTW.